trace_point.rb

```diff
@@ -167,15 +167,16 @@ def self.allow_reentry
   #   trace.enable    #=> true (previous state)
   #                   #   trace is still enabled
   #
-  # If a block is given, the trace will only be enabled within the scope of the
-  # block.
+  # If a block is given, the trace will only be enabled during the block call.
+  # block. If target and target_line are both nil, then target_thread will default
```

Comment:
A leftover `block.` here, feel free to squash my suggestion(s) in your commit(s).